frame/contracts/src/lib.rs

```diff
@@ -369,49 +369,44 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// Contract deployed by address at the specified address. \[deployer, contract\]
```

Comment:
@jacogr I've removed this `\[deployer, contract\]`, but is that used by `polkadot.js` currently for the field names?

Reply:
No, it not used at all - it purely appears in the md documentation generated from the metadata, aka https://polkadot.jso.org/docs/.

Reply:
You will find this annotations sprinkled everywhere. If moving to this field name syntax, I think you can clean them all up :)